src/main/java/guru/qa/allure/notifications/chart/Chart.java

```diff
@@ -19,14 +21,23 @@
 public class Chart {
 
     public static byte[] createChart(Base base) throws MessageBuildException {
+        return createChart(base, null);
+    }
+    public static byte[] createChart(Base base, TestOps testOps) throws MessageBuildException {
```

Comment:
the thing here is generic Chart starts "knowing" about TestOps, which is not good if low coupling is followed